node/sync.go

```diff
@@ -372,10 +376,14 @@ func (d *Pegnetd) ApplyTransactionBlock(sqlTx *sql.Tx, eblock *factom.EBlock) er
 		}
 
 		// No conversions in the batch, it can be applied immediately
-		if err = d.applyTransactionBatch(sqlTx, txBatch, nil, eblock.Height); err != nil && err != pegnet.InsufficientBalanceErr {
+		if err = d.applyTransactionBatch(sqlTx, txBatch, nil, eblock.Height); err != nil &&
+			err != pegnet.InsufficientBalanceErr && // Allowed Exception
+			err != pegnet.PFCTOneWayError { // Allowed Exception
 			return err
 		} else if err == pegnet.InsufficientBalanceErr {
 			d.Pegnet.SetTransactionHistoryExecuted(sqlTx, txBatch, -1)
+		} else if err == pegnet.PFCTOneWayError {
+			d.Pegnet.SetTransactionHistoryExecuted(sqlTx, txBatch, -2)
```

Comment:
technically, this error can only occur when there are transactions, so it would never happen here. this would be the place to throw something other than a pfctonewayerror so that it's detected

Reply:
Ahh because conversions cannot get here?
We should really find a way to use the same error switch statement vs copying it in both places.